internal/bminventory/inventory_test.go

```diff
@@ -6246,7 +6314,37 @@ var _ = Describe("infraEnvs", func() {
 				Expect(err).ToNot(HaveOccurred())
 				Expect(common.ImageTypeValue(i.Type)).To(Equal(models.ImageTypeFullIso))
 			})
+			It("Update additional trust bundle", func() {
+				var err error
+				mockInfraEnvUpdateSuccess()
 
+				err = db.Model(&common.InfraEnv{}).Where("id = ?", i.ID).Update("additional_trust_bundle", testCert).Error
+				Expect(err).ToNot(HaveOccurred())
+				reply := bm.UpdateInfraEnv(ctx, installer.UpdateInfraEnvParams{
+					InfraEnvID: *i.ID,
+					InfraEnvUpdateParams: &models.InfraEnvUpdateParams{
+						// New lines added before and after to test that they are ignored
+						AdditionalTrustBundle: fmt.Sprintf("\n%s\n\n", testCert2),
+					},
+				})
+				Expect(reply).To(BeAssignableToTypeOf(installer.NewUpdateInfraEnvCreated()))
+				i, err = bm.GetInfraEnvInternal(ctx, installer.GetInfraEnvParams{InfraEnvID: *i.ID})
+				Expect(err).ToNot(HaveOccurred())
+				Expect(i.AdditionalTrustBundle).To(Equal(testCert2))
+			})
+			It("Update additional trust bundle - invalid", func() {
+				err := db.Model(&common.InfraEnv{}).Where("id = ?", i.ID).Update("additional_trust_bundle", testCert).Error
+				Expect(err).ToNot(HaveOccurred())
+				reply := bm.UpdateInfraEnv(ctx, installer.UpdateInfraEnvParams{
+					InfraEnvID: *i.ID,
+					InfraEnvUpdateParams: &models.InfraEnvUpdateParams{
+						// New lines added before and after to test that they are ignored
+						AdditionalTrustBundle: "somejunk",
+					},
+				})
+				Expect(reply).To(BeAssignableToTypeOf(&common.ApiErrorResponse{}))
+				Expect(reply.(*common.ApiErrorResponse).StatusCode()).To(Equal(int32(http.StatusBadRequest)))
+			})
```

Comment:
An `It("Update additional trust bundle - unset", func() {` just below this one, and you will be covered for what Vadim pointed out